diff --git uc_option_image.install uc_option_image.install
index 7740772..5859b90 100644
--- uc_option_image.install
+++ uc_option_image.install
@@ -11,7 +11,7 @@
  */
 function uc_option_image_install() {
   $default_image_path = drupal_get_path('module', 'uc_option_image') . '/no_image.png';
-  $default_image_name = 'option_image_0_0_0';
+  $default_image_name = 'option_image_0_0_0',
   $file_path = db_result(db_query("SELECT fid from {files} WHERE filename = '%s' AND filepath = '%s'", $default_image_name, $default_image_path));
   // Make sure image doesn't already exist in the files table.
   if (!$file_path) {
